app/screens/channel/channel.android.js

```diff
@@ -74,14 +72,8 @@ export default class ChannelAndroid extends ChannelBase {
     }
 }
 
-const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
-    return {
-        backdrop: {
-            flex: 1,
-            backgroundColor: theme.centerChannelBg,
-        },
-        flex: {
-            flex: 1,
-        },
-    };
```

Comment:
Seems so obvious now to not have a background, on top of a background, on top of a background.